winlogbeat/etc/kibana/dashboard/Winlogbeat-Dashboard.json

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
Another solution would be to store the raw json file as it is exported by Kibana. The biggest advantage is that we don't have to maintain it.